usaspending_api/static_doc_files/js/jquery-2.2.0.min.js

```diff
@@ -1,4 +1,2 @@
-/*! jQuery v2.2.0 | (c) jQuery Foundation | jquery.org/license */
```

Comment:
Shouldn't the newer version of jQuery live in `jquery-3.4.1.min.js`?

Would need to update the filename src links in /usaspending_api/templates/index.html and /usaspending_api/templates/plate.html

Reply:
Concur